docs/pages/blog/mui-x-mid-v6-features.md

```diff
@@ -0,0 +1,130 @@
+
+This feature allows users to quickly filter the data without needing any additional menu and can be activated using the `unstable_headerFilters` prop.
+
+The new filter fields are displayed below the headers and are synchronized with the filter panel, but for those seeking more simplicity, disable the default filter panel using `disableColumnFilter` prop and set filters to use only the default operator.
+You can also customize the header filter cell for a specific column using the `renderHeaderFilter` method of `GridColDef`
+
+For more details on how to use and customize the filters, check out its [documentation page](https://mui.com/x/react-data-grid/filtering/header-filters/).
+
+### **Performance Enhancements: Smoother Scrolling and Expanding Rows**
+
+The Data Grid received significant performance improvements in MUI X v6, particularly concerning the data grid's scrolling efficiency and the expansion of children's rows.
+
+These performance upgrades are part of our ongoing commitment to provide end-users with a superior experience.
+We always strive to ensure that MUI X remains performant and efficient, particularly when dealing with large and complex data.
+
+## Charts - alpha version 🎁
+
+We are delighted to announce the upcoming expansion of MUI X– a brand-new suite of components specifically designed for building and easily customizing appealing charts.
+
+With MUI X Charts, you can choose from a wide range of chart types, including line charts, bar charts, pie charts, area charts, scatter plots, and more. Each chart type is thoughtfully crafted with attention to detail, ensuring that the visual representations are not only aesthetically pleasing but also highly effective in conveying complex data.
+
+And it ships with a couple of rad palettes, specially crafted by our designers!
+
+Checkout a video highlighting some of our Charts!
+
+<video width="400" height="400" preload="metadata" controls>
+  <source src="/static/blog/mui-x-mid-v6-features/charts_final_v1.mov" type="video/mp4">
+</video>
+
+As we progress toward the stable version, we're committed to enhancing the overall experience for developers and users, so your feedback is fundamental!
+
+We will also steadily expand our portfolio with new chart types; the Pie Chart is next on our list for the first stable version.
+However, if there's a specific chart visualization you'd like us to prioritize, we encourage you to upvote the respective issue on [GitHub](https://github.com/mui/mui-x/issues?q=is%3Aissue+is%3Aopen+label%3A%22component%3A+charts%22+label%3A%22waiting+for+%F0%9F%91%8D%22). Your input can directly influence our development schedule, so don't hesitate to let us know what matters most to you!
+
+[Get started with charts now!](https://mui.com/x/react-charts/)
+
+## Discord
+
+<img src="/static/blog/mui-x-mid-v6-features/discord.png" alt="" width="2400" height="600" style="margin-bottom: 16px;" />
```

Comment:
1. The link on the image is https://bit.ly/mui-discord-server which points to the old server

<img width="258" alt="Screenshot 2023-07-31 at 16 31 19" src="https://github.com/mui/material-ui/assets/3165635/c74d143b-9bc6-4ead-8d09-1119183a9ebe">

to remove or replace for https://mui.com/r/discord.

Reply:
2. The MUI logo on the image doesn't look like the production logo, to update:

<img width="217" alt="Screenshot 2023-07-31 at 16 35 59" src="https://github.com/mui/material-ui/assets/3165635/15e8c815-4f18-42b4-b25e-90be48bd6108">

vs.

<img width="124" alt="Screenshot 2023-07-31 at 16 33 19" src="https://github.com/mui/material-ui/assets/3165635/63f9b957-38a2-4e09-bccc-11a8cee9b52c">

https://www.figma.com/file/nNAatyLlRl3Tq4nGHgAmNh/Primitives?type=design&mode=design&t=6EIPm30cEcCC6g6b-0

Reply:
3. The dimension, it's pixelized on my screen

```suggestion
<img src="/static/blog/mui-x-mid-v6-features/discord.png" alt="" width="640" height="320" style="margin-bottom: 16px;" />
```

but likely better to have a higher resolution export.

Reply:
Ah yes, The images were taken from the old blog which had outdated images. Will update. Thanks for the catch!